src/gql/generated/types.ts

```diff
@@ -347,7 +347,6 @@ export type DispatcherSettingsInput = {
 };
 
 export enum DispatcherVersion {
-  Revised = "REVISED",
```

Comment:
i think you may want to leave this file unchanged, I think we typically don't regenerate this file until a backend change is fully merged